styles.css

```diff
@@ -0,0 +1,320 @@
+      "content-team"
+      "content-sponsors"
+      "footer";
+  }
+  
+  .wrapper > *:not(.content-why):not(.content-sponsors) {
+    padding-left: 50px;
+    padding-right: 50px;
+  }
+  
+  .nav {
+    grid-area: nav;
+    margin: 1em;
+  }
+  
+  .header-box {
+    grid-area: header;
+    border-bottom: 1px black solid;
+  }
+  
+  .content-quote:first-of-type {
+    grid-area: content-quote-1;
+  }
+  
+  .content-quote:nth-of-type(2) {
+    grid-area: content-quote-2;
+  }
+  
+  .content-why {
+    grid-area: content-why;
+    border-bottom: 1px black solid;
+  }
+  
+  .content-team {
+    grid-area: content-team;
+  }
+  .content-sponsors {
+    grid-area: content-sponsors;
+    border-bottom: 1px black solid;
+  }
+  .main-footer {
+    grid-area: footer;
+  }
+  
+  /* nav */
+  
+  .nav-box {
+    display: flex;
+    margin: 1em;
+    list-style: none;
+  }
+  
+  .nav-box > li {
+    padding-left: 15px;
+    padding-right: 15px;
+    padding-top: 10px;
+    padding-bottom: 10px;
+    border-radius: 2em;
+    border: 1px solid black;
+  }
+  
+  .nav-try {
+    margin-left: auto;
+    margin-right: 80px;
+  }
+  
+  /* header */
+  
+  .header-box {
+    display: flex;
+    flex-direction: column;
+    text-align: center;
+  }
+  
+  .header-box > div {
+    padding: 20px;
+  }
+  
+  .header-box div:last-of-type {
+    margin: 0;
+    min-height:15vh;
+    display:flex;
+    align-items:center;
+    justify-content:center;
+  }
+  
+  a.header-button {
+    font-size: 25px;
+    border: 2px solid silver;
+    padding: 1rem;
+    text-decoration:none;
+  }
+  
+  a.header-button:hover {
+    background-color: tomato;
+    color: white;
+  }
+  
+  /* content-quote */
+  
+  .content-quote {
+    display: flex;
+    flex-direction: column;
+    text-align: center;
+    margin: 20px;
+    font-style: italic;
+    border-bottom: 1px black solid;
+  }
+  
+  .content-quote blockquote {
+    background: #eee;
+    border-radius: 5px;
+    padding: 20px;
+    font-size: 20px;
+    line-height: 35px;
+  }
+  
+  .content-quote p {
+    margin-left: auto;
+    margin-right: 80px;
+    font-size: 25px;
+    padding-bottom: 40px;
+  }
+  
+  .content-quote p::before {
+    content: "—	"
+  }
+  
+  blockquote::before {
+    content: '\201C';
+  }
+  
+  blockquote::after {
+    content: '\201D';
+  }
+  /* content-why */
+  
+  .content-why {
+    display: flex;
+    flex-direction: column;
+    align-content: flex-start;
+  }
+  
+  ul.why-pictures-box {
+    padding: 0;
+    margin: 0;
+    list-style: none;
+    display: flex;
+    flex-direction: row;
+  }
+  
+  ul.why-pictures-box li {
+    flex: 1;
+    padding-right: 50px;
+    margin: 0;
+  }
+  
+  ul.why-pictures-box img {
+    float: left;
+    padding-right: 10px;
+  }
+  
+  ul.why-pictures-box p {
+    text-align: left;
+    padding-bottom: 80px;
+  }
+  
+  .content-why h2 {
+    text-align: center;
+    margin: 0;
+    padding-bottom: 30px;
+  }
+  
+  /* content-team */
```

Comment:
In this section, it might be helpful to use a Flexbox to get the text lined up under the images. A TA helped me with this section and we got it to work by making the unordered list (ul) a Flexbox with flex-direction: row and the list items (li) a Flexbox with flex-direction: column.